templates/docs/patterns/icons.md

```diff
@@ -9,7 +9,15 @@ context:
 
 <hr>
 
-Icons provide visual context and enhance usability, they can be added via an `<i>` element like so: `<i class="p-icon--{name}"></i>`
+Icons provide visual context and enhance usability, they can be added via an `<i>` element like so: `<i class="p-icon--{name}"></i>`.
+
+### Icons as links
```

Comment:
Looks good, but I think we should place it further down the page, after we show all the icons.

Reply:
I did do that initially - but then I looked at the paragraph above, near the top of the page:
> Icons provide visual context and enhance usability, they can be added via an <i> element like so: <i class="p-icon--{name}"></i>.

and it felt weird that the documentation for this alternate application was so far removed from this "standard" one. If I was coming to this page, I might miss that it was an option.